src/cloud/project/new-relic.md

```diff
@@ -69,40 +69,53 @@ runtime:
+-  For Starter accounts, you have a New Relic license key that supports up to three environments. You must add the key to your environment configurations manually. The license key is not pre-provisioned on Starter environments.
+
+### Configure New Relic for Starter environments
+
+For Starter environments, you must add the New Relic license key to the environment configuration to enable the New Relic integration. We recommend adding the key to the Staging and Production (master) environments and one other environment of your choice. Only the New Relic license key is required for configuration. You can find information about additional configuration options in the [New Relic reporting] topic in the _Magento User Guide_.
+
+{:.bs-callout-warning}
+Updating the environment configuration triggers a redeployment, which takes your site offline until deployment completes. For Production environments, we recommend completing this work during off-peak hours to avoid service disruptions. See [Working with variables].
 
 {:.procedure}
-To find your New Relic license key:
+Prerequisites
 
-1. Open your [account page](https://accounts.magento.cloud/user/).
+-  Log in credentials for the Magento Cloud project account page, or for the New Relic account associated with your project
+-  [Admin level access]({{site.baseurl}}/cloud/project/user-admin.html) to the Starter environments to configure, or credentials to access the [Magento Admin](https://docs.magento.com/m2/ce/user_guide/system/permissions.html) for the environment.
 
-1. On the _Projects_ tab, find your project.
+{:.procedure}
+To configure New Relic on a Starter environment:
 
-1. Click **View Details** to see the project infrastructure information.
+1. Find your New Relic license key from your Magento account page, or [from your New Relic account page]:
 
-1. Expand the **NewRelic Service** section to view the license key.
+   -  Open your [Magento account page].
 
-   ![Cloud project details page]({{ site.baseurl }}/common/images/cloud/cloud-project-details-new-relic.png){:width="650px"}
+   -  On the _Projects_ tab, find your project.
 
-1. Copy the license key.
+   -  Click **View Details** to see the project infrastructure information.
 
-{:.procedure}
-To add a license key to an environment:
+   -  Expand the **New Relic Service** section to view the license key.
 
-1. Change to the environment that needs the license key. For Starter accounts, you need to use the license key to set a variable in the environment.
+   -  Copy the license key.
 
-1. Set the variable.
+1. Add the New Relic license key to an environment using an environment variable, or [add it from the Magento Admin]:
```

Comment:
```suggestion
1. Add the New Relic license key to an environment using the magento-cloud CLI, or [add it from the Magento Admin]:
```

Need to be specific. Saying environment variable may make some people think about the .magento.env.yaml file. or the Project Web UI variables list.